.gitignore

```diff
@@ -56,3 +56,4 @@ typechain/
 
 # Go bindings generator
 pkg/chain/*/*/gen/_contracts
+pkg/chain/*/*/gen/_address
```

Comment:
Does it work for you? I see all `gen/_address` changes in my index.

Reply:
It does not! 😠

Reply:
So we can leave the `.gitignore` as is, and all we need to do is to `rm pkg/chain/ethereum/threshold/gen/_address/TokenStaking` and the rest of the files holding the address and commit this change.

These files will be generated during `go generate ./...` but not included in the git diff.